src/ts/components/volumeslider.ts

```diff
@@ -70,19 +70,29 @@ export class VolumeSlider extends SeekBar {
     // Update the volume slider marker when the player resized, a source is loaded and player is ready,
     // or the UI is configured. Check the seekbar for a detailed description.
     player.addEventHandler(player.EVENT.ON_PLAYER_RESIZE, () => {
-      this.refreshPlaybackPosition();
+      this.refreshPlaybackPosition(true);
     });
     player.addEventHandler(player.EVENT.ON_READY, () => {
-      this.refreshPlaybackPosition();
+      this.refreshPlaybackPosition(true);
     });
     uimanager.onConfigured.subscribe(() => {
-      this.refreshPlaybackPosition();
+      this.refreshPlaybackPosition(true);
     });
 
     // Init volume bar
     volumeChangeHandler();
   }
 
+  /**
+   * Refreshes the playback position. This is needed because subclassing {@link seekBar} is a hack
```

Comment:
"Refreshes the volume slider's position. This is needed because we want to avoid the default behavior of {@link SeekBar.onShowEvent}, which is to refresh the slider position when it appears because that screws up our volume slider's position. To avoid it we override this method and add a conditional flag that defaults to `false`, avoiding the update from all calls coming from the base class ({@link SeekBar}). We can still request updates if we supply `true`."

Reply:
Lol, there is a GitHub user called link. So everytime someone posts some JavaDoc/JSDoc/etc... without a surrounding code block with an @link tag in it, they receive a notification. That must be a lot of notifications 😁

Reply:
Haha poor guy hope he has good filtering setup.